src/test/java/it/pagopa/pn/bff/pnclient/userattributes/PnUserAttributesClientImplTest.java

```diff
@@ -38,6 +38,40 @@ class PnUserAttributesClientImplTest {
     @MockBean(name = "it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.api.LegalApi")
     private LegalApi legalApi;
 
+
+    @Test
+    void getPgConsentByType() throws RestClientException {
+        Consent consent = consentsMock.getTosConsentResponseMock();
```

Comment:
create a getPgTosConsentResponseMock that the return the consent with type TOS_DEST_B2B

Reply:
create